index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel = "stylesheet"
+type = "text/css"
+href = "css/style.css" />
+ <!-- <script src = "js/general.js" -->
+</head>
+<body>
+<script src = "js/general.js"></script> 
+<div class = "nav">
+    <ul>
+        <li id = "logo"> I-Reporter</li>
+        <li id = "topnav">home</li>
+        <li id = "topnav"><button onclick="Signup()">signup</button></li>
+        <li id = "topnav"><button onclick="Login()">login</button></li>
+        <li id = "topnav">about us</li>
+     
+        
+        
+    </ul>
+   
+</div>
+<div class = "main" > 
+    
+    <div id ="main-message" style="background-image: url(img/cor.jpg);">
+    <div id ="message">REPORT CORRUPTION</div>
+    <div id ="messag">dont be silent</div>
+    <div id = "login">
+        
+        <p>LOGIN</p>
+        <p>username</p>
+        <p><input type = "text" name ="enter username" required></p>
+        <p>password</p>
+        <p><input type ="text" name = "enter password" required></p>
+
+        <p><button type ="submit">submit</button></p>
+        <p>If you dont have an account signup here</p>
+    </div>
+    <div id = "signup">
+        
+            <p>SIGNUP HERE</p>
+            <p>first name</p>
+            <p><input type = "text" name ="enter first name" required></p>
```

Comment:
I think you shouldn't nest your <input> elements/tags in <p> tags. They can be on their own.